File: cycle_menu/src/main/java/com/cleveroad/sy/cyclemenuwidget/CycleMenuItem.java
package com.cleveroad.sy.cyclemenuwidget;

import android.graphics.drawable.Drawable;

/**
 * Model class for menu items
 */
public class CycleMenuItem {
    private Drawable mIcon;
    private int mId;
    private int mColor;

    public CycleMenuItem(int id, Drawable icon) {
        mId = id;
        mIcon = icon;

    }

    public CycleMenuItem(int color) {
        mColor = color;
    }

    public int getId() {
        return mId;
    }

    public Drawable getIcon() {
        return mIcon;
    }

    public int getColor() {
        return mColor;
    }
}

File: cycle_menu/src/main/java/com/cleveroad/sy/cyclemenuwidget/CycleMenuItem.kt
package com.cleveroad.sy.cyclemenuwidget

import android.graphics.drawable.Drawable

/**
 * Model class for menu items
 */
data class CycleMenuItem @JvmOverloads constructor(
    var icon: Drawable? = null,
    var id: Int = -1,
    var color: Int = -1
)
